test/automated/ansible/group_vars/localhost/main.yml

```diff
@@ -70,20 +70,20 @@ instances:
     platform: "linux"
     python_interpreter: "/usr/bin/python3"
     launch_template: "LaunchTemplateId=lt-0b00afb3f5110a0e6,Version=3"
-  - ami: "ami-0a9e8a2a76b55f509"
-    type: "t4g.small"
-    name: "arm64:ubuntu21.04"
-    username: "ubuntu"
-    platform: "linux"
-    python_interpreter: "/usr/bin/python3"
-    launch_template: "LaunchTemplateId=lt-0b00afb3f5110a0e6,Version=3"
-  - ami: "ami-0e8eba75fce52c37c"
-    type: "t4g.small"
-    name: "arm64:ubuntu20.10"
-    username: "ubuntu"
-    platform: "linux"
-    python_interpreter: "/usr/bin/python3"
-    launch_template: "LaunchTemplateId=lt-0b00afb3f5110a0e6,Version=3"
+#  - ami: "ami-0a9e8a2a76b55f509"
```

Comment:
I would add a comment to explain why this was commented (and if it's something that will not come back I would delete it :) )